[2025-02-13T02:06:30.405+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-13T02:06:30.435+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_for_dbt_denormalize.Upload_data_to_s3 manual__2025-02-12T21:06:05-05:00 [queued]>
[2025-02-13T02:06:30.457+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_for_dbt_denormalize.Upload_data_to_s3 manual__2025-02-12T21:06:05-05:00 [queued]>
[2025-02-13T02:06:30.459+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-02-13T02:06:30.484+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): Upload_data_to_s3> on 2025-02-13 02:06:05+00:00
[2025-02-13T02:06:30.495+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1329) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-13T02:06:30.498+0000] {standard_task_runner.py:72} INFO - Started process 1331 to run task
[2025-02-13T02:06:30.499+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_for_dbt_denormalize', 'Upload_data_to_s3', 'manual__2025-02-12T21:06:05-05:00', '--job-id', '260', '--raw', '--subdir', 'DAGS_FOLDER/denormalize_data_gad.py', '--cfg-path', '/tmp/tmpldby7_i4']
[2025-02-13T02:06:30.501+0000] {standard_task_runner.py:105} INFO - Job 260: Subtask Upload_data_to_s3
[2025-02-13T02:06:30.579+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_for_dbt_denormalize.Upload_data_to_s3 manual__2025-02-12T21:06:05-05:00 [running]> on host f917fd63a571
[2025-02-13T02:06:30.708+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_for_dbt_denormalize' AIRFLOW_CTX_TASK_ID='Upload_data_to_s3' AIRFLOW_CTX_EXECUTION_DATE='2025-02-13T02:06:05+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-12T21:06:05-05:00'
[2025-02-13T02:06:30.709+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-13T02:06:41.155+0000] {logging_mixin.py:190} INFO - uploaded file sub.txt
[2025-02-13T02:06:53.608+0000] {logging_mixin.py:190} INFO - uploaded file pre.txt
[2025-02-13T02:07:45.545+0000] {logging_mixin.py:190} INFO - uploaded file num.txt
[2025-02-13T02:07:49.244+0000] {logging_mixin.py:190} INFO - uploaded file tag.txt
[2025-02-13T02:07:49.489+0000] {logging_mixin.py:190} INFO - uploaded file readme.htm
[2025-02-13T02:07:49.642+0000] {python.py:240} INFO - Done. Returned value was: success
[2025-02-13T02:07:50.123+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-13T02:07:50.133+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=dag_for_dbt_denormalize, task_id=Upload_data_to_s3, run_id=manual__2025-02-12T21:06:05-05:00, execution_date=20250213T020605, start_date=20250213T020630, end_date=20250213T020750
[2025-02-13T02:07:50.564+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-13T02:07:50.746+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-13T02:07:50.752+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
